### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by stratusjerry**
fixes typos

### CODE REVIEW

File: administration/monitoring.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ The following definitions are key to understand:
...
-| process_start_time_seconds             |                                                 | The Unix Epoch time stamp for when Fluent Bit started..                                                                                                                                                                                                                                      | guage   | seconds |
+| process_start_time_seconds             |                                                 | The Unix Epoch time stamp for when Fluent Bit started..                                                                                                                                                                                                                                      | gauge   | seconds |
```
Comment (cnorris-cs) [inline administration/monitoring.md:121]:
```suggestion
| process_start_time_seconds             |                                                 | The Unix Epoch time stamp for when Fluent Bit started. | gauge | seconds |
```

Comment (cnorris-cs) [reply administration/monitoring.md:121]: I added a suggestion that gets rid of both an additional period, as well as additional unneeded space in the table.

**Comment by lecaros** [comment]
Hi @stratusjerry thanks for your contribution.
Before we can merge this, please review our [Contributing guidelines](https://github.com/fluent/fluent-bit-docs/blob/master/CONTRIBUTING.md) and comply with them. In specific, please review the signing-off commits and the commit messages.

File: administration/monitoring.md (hunk 1 above)
Comment (stratusjerry) [reply administration/monitoring.md]: thanks, I took your changes and rebased/force pushed

**Review by cnorris-cs — APPROVED** [review]

File: administration/backpressure.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ The setting behaves similarly to the above scenario with `Mem_Buf_Limit` when th
 
 When (default) `storage.pause_on_chunks_overlimit` is disabled, the input will not pause when the memory limit is reached. Instead, it will switch to only buffering logs in the filesystem. The disk spaced used for filesystem buffering can be limited with `storage.total_limit_size`.
 
-Please consule the [Buffering & Storage](buffering-and-storage.md) docs for more information.
+Please consult the [Buffering & Storage](buffering-and-storage.md) docs for more information.
```
Comment (cnorris-cs) [inline administration/backpressure.md:47]:
```suggestion
See the [Buffering & Storage](buffering-and-storage.md) docs for more information.
```

**Pull request merged**